pkg/apis/kubermatic/v1/settings.go

```diff
@@ -169,6 +175,17 @@ type ProviderConfiguration struct {
 	VMwareCloudDirector VMwareCloudDirectorSettings `json:"vmwareCloudDirector,omitempty"`
 }
 
+type WebTerminalOptions struct {
+	// Enabled enables the Web Terminal feature for the user clusters.
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
Question (not a merge blocker, just curious to preserve the reasoning for the future): Why is this a bool pointer over a normal bool (like `enableInternetAccess`)? Is there a difference between unset (`nil`) and `false` we want to make available here?

Reply:
I detest pointers to booleans TBH (:D) but this was done intentionally so that we can check if this field was explicitly set or not; to differentiate between explicit zero and not specified. This is only required to sanely migrate the `EnableWebTerminal` value.

Reply:
Thanks, makes sense!